rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -280,4 +305,4 @@ sudo systemctl disable lnd_admin
 systemctl daemon-reload
 
 # Sync FS
-sync
\ No newline at end of file
+sync
```

Comment:
Add `systemctl enable whirlpool`, so that the GUI can be connected in the first step.